src/FlowtypeRunner.js

```diff
@@ -0,0 +1,48 @@
+const { exec } = require('child_process');
+const path = require('path');
+const { pass, fail } = require('create-jest-runner');
+
+class FlowtypeRunner {
+  constructor(globalConfig) {
+    this.globalConfig = globalConfig;
+  }
+
+  runTests(tests, watcher, onStart, onResult, onFailure, options) {
```

Comment:
can you make these keyword args? `runTests({ tests, watcher, onStart, onResult, onFailure, options })`

Reply:
This method is being called by jest so I can't play with it. Many of those arguments are actually irrelevant for my implementation but I still need to show them to make it clear that I'm not the one to define the api...
(I will add a comment) 👍